docs/data-management.rst

```diff
@@ -226,6 +232,87 @@ Typically an administrator works on behalf of other users in a group where the a
 
 #. When creating new Datasets or Projects during the move, note that these containers will belong to the owner of the data, not yourself. Also the links between the new containers and the moved data will belong to the owner of the data. This should help to facilitate a smooth workflow, retaining the data handling possibilities such as reorganizing the data, renaming the containers you created for them etc. for the owner of the data. 
 
+.. _Duplicatecli:
+
+*Command Line: Duplicating objects*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can duplicate objects in OMERO. The functionality is available only on the CLI for now. The duplication creates a fully independent duplicate of the duplicated object and, when desired, the objects linked to the duplicated object as well. When the duplicate is later deleted, it will not have an influence on the original, which stays preserved. Similarly, when the original is later deleted, the duplicate stays preserved as well.
```

Comment:
```suggestion
You can duplicate objects in OMERO. The functionality is available only on the CLI for now. The duplication creates a full copy of the objects as if they were created independently. When desired, the newly duplicated objects can additionally be linked to other objects. When the duplicate is later deleted, it will not have an influence on the original, which stays preserved. Similarly, when the original is later deleted, the duplicate stays preserved as well.
```